<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>170</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>170</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>360</width>
    <height>170</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>pickWalkAuthor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QRadioButton" name="navigateRadioBTN">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Navigate</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="createRadioBTN">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>-4</y>
      <width>82</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Create</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>20</y>
      <width>421</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="medPushBTN">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Nothing selected</string>
    </property>
   </widget>
   <widget class="QPushButton" name="upPushBTN">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>blank</string>
    </property>
   </widget>
   <widget class="QPushButton" name="downPushBTN">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>blank</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rightPushBTN">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>80</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>blank</string>
    </property>
   </widget>
   <widget class="QPushButton" name="leftPushBTN">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>blank</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>navigateRadioBTN</sender>
   <signal>clicked(bool)</signal>
   <receiver>medPushBTN</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>153</x>
     <y>10</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>createRadioBTN</sender>
   <signal>clicked(bool)</signal>
   <receiver>medPushBTN</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>10</y>
    </hint>
    <hint type="destinationlabel">
     <x>155</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
